scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -31,14 +31,20 @@
 ACCEPTED_ENCODINGS: list[bytes] = [b"gzip", b"deflate"]
 
 try:
-    try:
-        import brotli  # noqa: F401
-    except ImportError:
-        import brotlicffi  # noqa: F401
```

Comment:
I think we can restore brotlicffi compatibility after https://github.com/python-hyper/brotlicffi/pull/207 is released, but it's fine to drop it now.